src/GraphQL.ApiTests/net50/GraphQL.approved.txt

```diff
@@ -2207,6 +2207,12 @@ namespace GraphQL.Types
         public override object? ParseLiteral(GraphQLParser.AST.GraphQLValue value) { }
         public override object? ParseValue(object? value) { }
     }
+    public class GenericGraphTypeFactory<TGraphType> : GraphQL.Types.IGraphTypeFactory<TGraphType>
```

Comment:
@PSanetra run `GraphQL.ApiTests.csproj` tests locally to update the approved API

Reply:
Done :slightly_smiling_face: